codebusters-ca/react-firebase-starter
# Remove Firebase config, update setup script

## Description

This PR removes all Firebase config values from the code and replaces them with environment variables to prevent accessing any existing Firebase project by default.

It also updates the setup script to automate most setup steps for better DX.

## Pull request type

- [x] Bugfix:bug:
- [ ] Code style update (formatting, renaming):art:
- [x] Feature:sparkles:
- [ ] Refactoring (no functional changes, no api changes):hammer:
- [ ] Documentation Update:books:

## Related Issue

Closes #25, #26 

## Testing Steps / QA Criteria

* Log out of the Firebase CLI: `firebase logout`
* In a new directory, run a smoke test by following the instructions from README
* Run the emulators with `npm run emulators`, then in another terminal start the app with `npm start`
* In the browser, you should see `Hello from Firebase Emulator`
